deps/wxWidgets/wxWidgets.cmake

```diff
@@ -1,4 +1,8 @@
-set(_wx_git_tag v3.1.5)
+# set(_wx_git_tag v3.1.5) # Does not link -ldep-name-NOTFOUND (see deps/build/dep_wxWidgets-prefix/src/dep_wxWidgets-build/wx-config --libs all | grep NOTFOUND)
+# set(_wx_git_tag v3.1.4-patched) # Links, but Bambu's slic3r_gui depends on wxWebView methods in >=3.1.5 like SetUserAgent
+# set(_wx_git_tag v3.1.6) # wx includes nanosvg, which creates link conflicts (multiple definition) with slic3r which also uses nanosvg, wx nanosvg can't be disabled
+# set(_wx_git_tag v3.1.7) # wx includes nanosvg, which creates link conflicts ( multiple definition) with slic3r which also uses nanosvg, segfaults if nanosvg disabled
+set(_wx_git_tag v3.2.0) # Starts, but gives unhandled AppConfig::Save() exception
```

Comment:
we have submitted a fix for the wxWidgets compiling under linux before,
it should fixed the '-ldep-name-NOTFOUND' issue for v3.1.5
you can try it and remove the changes of wxWidgets

Reply:
I reverted my commit locally and wx built with Gerritt's change to the wxWidgets.cmake...

However, the slicer failed at linking time, haven't had a chance to dig into it. 

```
[100%] Built target libslic3r_gui                       Consolidate compiler generated dependencies of target BambuStudio                                               [100%] Linking CXX executable bambu-studio              /usr/bin/ld: ../../deps/build/destdir/usr/local/lib/libTKService.a(Font_FontMgr.cxx.o): undefined reference to symbol 'FcStrListNext'                                   /usr/bin/ld: /lib/x86_64-linux-gnu/libfontconfig.so.1: error adding symbols: DSO missing from command line      collect2: error: ld returned 1 exit status              make[3]: *** [src/CMakeFiles/BambuStudio.dir/build.make:201: src/bambu-studio] Error 1                          make[2]: *** [CMakeFiles/Makefile2:502: src/CMakeFiles/BambuStudio.dir/all] Error 2                             make[1]: *** [CMakeFiles/Makefile2:509: src/CMakeFiles/BambuStudio.dir/rule] Error 2                            make: *** [Makefile:218: BambuStudio] Error 2
```

I merged the other PR locally and made it require libfreetype-dev for Linux as well as Mac but still get the same error.

Reply:
this is a new issue which is caused by freetype used by OCCT in recent commits.

could you also add following changes for a try:
-------------------------------------------------------------------------------------
$ git diff ./src/libslic3r/CMakeLists.txt
diff --git a/src/libslic3r/CMakeLists.txt b/src/libslic3r/CMakeLists.txt
index c6f01a65d..268a6378e 100644
--- a/src/libslic3r/CMakeLists.txt
+++ b/src/libslic3r/CMakeLists.txt
@@ -454,7 +454,7 @@ set(OCCT_LIBS
     TKernel
 )

-if(APPLE)
+if(NOT WIN32)
     target_link_libraries(libslic3r freetype)
 endif()

Reply:
Unfortunately the libslicer cmake change to NOT WIN32 didn't make a difference...

![Screenshot_20220907-004728.jpg](https://user-images.githubusercontent.com/4991612/188791184-d5e2345c-2242-4c91-9356-06279ad1f717.jpg)

Reply:
Good news, it built... still segfaults but this is progress